core/modules/comment/comment.module

```diff
@@ -2126,7 +2126,7 @@ function comment_form_validate($form, &$form_state) {
         ->execute()
         ->fetchField();
       if ($taken) {
-        form_set_error('name', t('The name you used belongs to a registered user.'));
+        form_set_error('name', t('The name you used is a registered username. If it belongs to you then login to post. If not then use another name.'));
```

Comment:
Thanks for working on this @shruti1421 🙏🏼 ...please add commas to the if clauses, like so:

```suggestion
        form_set_error('name', t('The name you used is a registered username. If it belongs to you, then login to post. If not, then use another name.'));
```

See https://www.enago.com/academy/ifthen-using-conditional-sentences-in-academic-writing

> Use a comma to separate the if clause from the main clause when the if clause comes first.